docs_user/modules/proc_pulling-configuration-from-a-tripleo-deployment.adoc

```diff
@@ -58,12 +65,109 @@ services:
 +
 Repeat this step for each {OpenStackShort} service that you want to disable or enable.
 
+Os-diff can pull configuration or command output from non-containerized servies such as the ovs-external-ids:
+
+----
+services:
+  ovs_external_ids:
+    hosts:
+      - standalone
+    service_command: "ovs-vsctl list Open_vSwitch . | grep external_ids | awk -F ': ' '{ print $2; }'"
+    cat_output: true
+    path:
+      - ovs_external_ids.json
+    config_mapping:
+      ovn-bridge-mappings: edpm_ovn_bridge_mappings
+      ovn-bridge: edpm_ovn_bridge
+      ovn-encap-type: edpm_ovn_encap_type
+      ovn-match-northd-version: ovn_match_northd_version
+      ovn-monitor-all: ovn_monitor_all
+      ovn-remote-probe-interval: edpm_ovn_remote_probe_interval
+      ovn-ofctrl-wait-before-clear: edpm_ovn_ofctrl_wait_before_clear
+----
+
+This service is not an Openstack service executed in a container, so the description and the behavior is different.
+It's important to correctly configure an SSH config file or equivalent for non standard services such as OVS.
+The ovs_external_ids is not a service which runs in a container and the ovs data is stored
+on each host of our cloud: controller_1/controller_2/...
+
+The hosts key in the config.yaml will allow os-diff to loop over all hosts specified the output of the command, or the file or data that you need to pull from our deployment in order to compare it later:
```

Comment:
"... will allow os-diff to loop over all hosts specified the output of the command" -- doesn't make sense i think?